src/TodoList.jsx

```diff
@@ -0,0 +1,35 @@
+const todoList = [
+
+  {
+    id: 1,
+    title: "Complete Hawk assignment"
+  },
+  {
+    id: 2,
+    title: "Complete Horse assignment"
+  },
+  {
+    id: 3,
+    title: "Study English"
+  },
+  {
+    id: 4,
+    title: "Check university tasks"
+  },
+
+];
+
+export default function TodoList() {
+  
+  return (
+    <>
```

Comment:
You don't need these fragments here since you already have a top level element here (The <ul> wrapping list element) so you can remove the <></>